src/libstd/sys/unix/ext/net.rs

```diff
@@ -1408,6 +1408,32 @@ impl UnixDatagram {
         self.0.write(buf)
     }
 
+    /// Like `send`, except that it sends from a slice of buffers.
+    ///
+    /// The peer address may be set by the `connect` method, and this method
+    /// will return an error if the socket has not already been connected.
+    ///
+    /// On success, returns the number of bytes written.
+    ///
+    /// # Examples
+    ///
+    /// ```no_run
+    /// use std::os::unix::net::UnixDatagram;
+    /// use std::io::IoSlice;
+    ///
+    /// fn main() -> std::io::Result<()> {
+    ///     let sock = UnixDatagram::unbound()?;
+    ///     sock.connect("/some/sock").expect("Couldn't connect");
+    ///     let bufs = [IoSlice::new(b" "), IoSlice::new(b"hello world")];
+    ///     sock.send_vectored(&bufs).expect("send_vectored function failed");
+    ///     Ok(())
+    /// }
+    /// ```
+    #[stable(feature = "unix_socket", since = "1.10.0")]
```

Comment:
What is the correct way to specify this attribute? I didn't find the relevant documentation.